### DISCUSSION TITLE
Can theme fonts be utilized through XML styles?

### DISCUSSION THREAD

**Discussion by hshoja** [dolanmiu/docx#2323] [Q&A] [answered]
Even though I'm utilizing external styles, it appears that the default font styles are not being sourced from the theme. Is it possible to utilize them instead?

```ts
new d.Document({
    externalStyles: styles,
})
```

<img width="445" alt="Screenshot 2023-09-04 at 12 30 02" src="https://github.com/dolanmiu/docx/assets/14098200/f293a61d-8533-476f-b250-10bc030f5827">

**Comment by hshoja** [comment, marked as answer]
It appears that there is an issue when selecting a font from either `Theme fonts` or `Theme color`, as it does not work as intended. Therefore, if we wish to define a style and use it as an XML style, it must be a non-theme one. 

Would it be okay if I added some description about this to the documentation, @dolanmiu?

**Reply by dolanmiu** [reply]
Sure yes please, if you can update the documentation that would be great
